client/session-timeout.ts

```diff
@@ -136,25 +136,19 @@ export default class SessionTimeout {
   }
 
   keyDownEventListener = (event) => {
-    switch (event.key) {
-      case 'Tab':
-        event.preventDefault();
-        if (!this.modalElement.contains(document.activeElement) || this.focusableElements.length === 1) {
-          this.firstFocusableElement.focus();
-          break;
-        }
-        if (event.shiftKey) {
-          if (document.activeElement === this.firstFocusableElement) {
-            this.lastFocusableElement.focus();
-          }
-        } else {
-          if (document.activeElement === this.lastFocusableElement) {
-            this.firstFocusableElement.focus();
-          }
-        }
-        break;
-      default:
-        break;
+    if (event.key === 'Tab') {
+      event.preventDefault();
+      if (!this.modalElement.contains(document.activeElement) || this.focusableElements.length === 1) {
+        this.firstFocusableElement.focus();
+      }
+    } else if (event.shiftKey) {
+      if (document.activeElement === this.firstFocusableElement) {
+        this.lastFocusableElement.focus();
+      }
+    } else {
+      if (document.activeElement === this.lastFocusableElement) {
+        this.firstFocusableElement.focus();
+      }
```

Comment:
```suggestion
    if (event.key === 'Tab') {
      event.preventDefault();
      if (!this.modalElement.contains(document.activeElement) || this.focusableElements.length === 1) {
        this.firstFocusableElement.focus();
      }
      if (event.shiftKey) {
        if (document.activeElement === this.firstFocusableElement) {
          this.lastFocusableElement.focus();
        }
      } else {
        if (document.activeElement === this.lastFocusableElement) {
          this.firstFocusableElement.focus();
      }
```

`if(event.shiftkey) {...} 
else { ... }`
goes inside first `if(event.key === 'Tab')`